Comp/r-sas_ci_for_prop.qmd

```diff
@@ -6,37 +6,37 @@ execute:
 
 ## Introduction
 
-There are many different methods of calculating a confidence interval for a proportion (percentage).
+There are several ways to calculate a confidence interval (CI) for a proportion. You need to select the method based on if you have a 1 sample proportion (1 proportion calculated from 1 group of subjects), or if you have 2 samples and you want a CI for the difference in the 2 proportions. The difference in proportion can come from either 2 independent samples (different subjects in each of the 2 groups), or can be matched (the same subject with 1 result in 1 group and 1 result in the other group \[paired data\]).
```

Comment:
Do you want the slashes here?

Reply:
no... issue with visual / source again :(